drivers/pinctrl/pinctrl_sam.c

```diff
@@ -71,6 +71,10 @@ static void pinctrl_configure_pin(pinctrl_soc_pin_t pin)
 	if (port_func == SAM_PINMUX_FUNC_periph) {
 		soc_pin.flags |= (SAM_PINMUX_PERIPH_GET(pin)
 				  << SOC_GPIO_FUNC_POS);
+	} else if (port_func == SAM_PINMUX_FUNC_wakeup) {
+		soc_pin.flags |= (SAM_PINMUX_PERIPH_GET(pin)
+				  << SOC_GPIO_FUNC_POS)
```

Comment:
nit: maybe you could have an intermediate macro for `SAM_PINMUX_PERIPH_GET(pin) << SOC_GPIO_FUNC_POS)`? It's used  a couple time and the indentation is awful, would save a couple of parenthesis as well....

Reply:
I'll keep as is for the moment.